oaknational/Oak-Web-Application
# Refactor/use cohort to determine what is new lesson pages - LESQ-524

## Description

Music year: 1978

- Adds cohort to lesson overview
- Filters out legacy data a query level
- Changes 'new' tag and if we show share button from cohort.



## How to test

1. Go to https://deploy-preview-2214--oak-web-application.netlify.thenational.academy
Lesson overview page - everything should work as before.
## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
